src/test/kotlin/org/dif/SignedMessageTest.kt

```diff
@@ -0,0 +1,113 @@
+package org.dif
+
+import com.nimbusds.jose.util.JSONObjectUtils
+import org.dif.crypto.parse
+import org.dif.message.Message
+import org.dif.mock.DIDDocResolverMock
+import org.dif.mock.SecretResolverMock
+import org.dif.model.PackSignedParams
+import org.dif.model.UnpackParams
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class SignedMessageTest {
+    data class TestVector(val from: String, val expected: String)
+
+    companion object {
+        const val ALICE_DID = "did:example:alice"
+        const val BOB_DID = "did:example:bob"
+
+        private const val ID = "1234567890"
+        private const val TYPE = "http://example.com/protocols/lets_do_lunch/1.0/proposal"
+        private val BODY = mapOf("messagespecificattribute" to "and its value")
+
+        val PLAINTEXT_MESSAGE = Message.builder(ID, BODY, TYPE)
```

Comment:
Not critical now, rather for future: I believe we will re-use the Plaintext in all tests against test vectors, so may make sense to move it into a common place.